### ISSUE TITLE
oh no cybergrind uploads

### ISSUE THREAD

**Issue by Neubulae** [xzxADIxzx/Join-and-kill-em-together#91]
i am now have cheat what shoul di do
In plain English: Cybergrind will upload score obtained under multiplayer JAKET, which is not forseen and not what I wanted in the first place.

**Comment by xzxADIxzx** [comment]
Multiplayer Cyber Grind hasn't been uploading the score until the latest update that broke the entire mod. Just wait for the next update :3

**Comment by Neubulae** [comment]
Just asking, I see you've been busy refactoring the mod, is there any ETA of this new release?

**Comment by xzxADIxzx** [comment]
Tbh, no

**Comment by Neubulae** [comment]
Ah OK

**Comment by kierkat10** [comment]
> Just asking, I see you've been busy refactoring the mod, is there any ETA of this new release?

Yesterday he said, in exact words: "I have reworked the sync of 0-1 /// 4-3, so the update is 80% done and will be released somewhere this month (I hope)."

**Comment by xzxADIxzx** [comment]
So uh, there are a few things I still need to do, but mostly the update is done
